* Add support for raptor2
* remove raptor references from configure.ac's that don't need/use it anywhere in
  it's corresponding source code.

TODO: maintain compatibility with raptor(1) too?

diff -up sonic-visualiser-1.8/sonic-visualiser/configure.ac.raptor2 sonic-visualiser-1.8/sonic-visualiser/configure.ac
--- sonic-visualiser-1.8/sonic-visualiser/configure.ac.raptor2	2011-04-08 10:56:20.000000000 -0500
+++ sonic-visualiser-1.8/sonic-visualiser/configure.ac	2011-07-31 20:32:32.494052584 -0500
@@ -82,7 +82,7 @@ SV_MODULE_REQUIRED([samplerate],[sampler
 SV_MODULE_REQUIRED([vamp],[vamp >= 2.1],[vamp/vamp.h],[],[])
 SV_MODULE_REQUIRED([vamphostsdk],[vamp-hostsdk >= 2.0],[vamp-hostsdk/PluginLoader.h],[],[])
 SV_MODULE_REQUIRED([rubberband],[rubberband],[rubberband/RubberBandStretcher.h],[rubberband],[rubberband_new])
-SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
+#SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
 SV_MODULE_REQUIRED([rasqal],[rasqal >= 0.9.19],[rasqal/rasqal.h],[rasqal],[rasqal_new_world])
 SV_MODULE_REQUIRED([redland],[redland >= 1.0.10],[librdf.h],[rdf],[librdf_new_world])
 
diff -up sonic-visualiser-1.8/svapp/configure.ac.raptor2 sonic-visualiser-1.8/svapp/configure.ac
--- sonic-visualiser-1.8/svapp/configure.ac.raptor2	2011-04-08 10:56:48.000000000 -0500
+++ sonic-visualiser-1.8/svapp/configure.ac	2011-07-31 20:32:57.720737209 -0500
@@ -82,7 +82,7 @@ SV_MODULE_REQUIRED([samplerate],[sampler
 SV_MODULE_REQUIRED([vamp],[vamp >= 2.1],[vamp/vamp.h],[],[])
 SV_MODULE_REQUIRED([vamphostsdk],[vamp-hostsdk >= 2.0],[vamp-hostsdk/PluginLoader.h],[],[])
 SV_MODULE_REQUIRED([rubberband],[rubberband],[rubberband/RubberBandStretcher.h],[rubberband],[rubberband_new])
-SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
+#SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
 SV_MODULE_REQUIRED([rasqal],[rasqal >= 0.9.19],[rasqal/rasqal.h],[rasqal],[rasqal_new_world])
 SV_MODULE_REQUIRED([redland],[redland >= 1.0.10],[librdf.h],[rdf],[librdf_new_world])
 
diff -up sonic-visualiser-1.8/svcore/configure.ac.raptor2 sonic-visualiser-1.8/svcore/configure.ac
--- sonic-visualiser-1.8/svcore/configure.ac.raptor2	2011-04-08 10:56:35.000000000 -0500
+++ sonic-visualiser-1.8/svcore/configure.ac	2011-07-31 20:33:31.846310577 -0500
@@ -82,7 +82,7 @@ SV_MODULE_REQUIRED([samplerate],[sampler
 SV_MODULE_REQUIRED([vamp],[vamp >= 2.1],[vamp/vamp.h],[],[])
 SV_MODULE_REQUIRED([vamphostsdk],[vamp-hostsdk >= 2.0],[vamp-hostsdk/PluginLoader.h],[],[])
 SV_MODULE_REQUIRED([rubberband],[rubberband],[rubberband/RubberBandStretcher.h],[rubberband],[rubberband_new])
-SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
+SV_MODULE_REQUIRED([raptor2],[raptor2 >= 2.0.0],[raptor.h],[raptor],[raptor_start_parse])
 SV_MODULE_REQUIRED([rasqal],[rasqal >= 0.9.19],[rasqal/rasqal.h],[rasqal],[rasqal_new_world])
 SV_MODULE_REQUIRED([redland],[redland >= 1.0.10],[librdf.h],[rdf],[librdf_new_world])
 
diff -up sonic-visualiser-1.8/svgui/configure.ac.raptor2 sonic-visualiser-1.8/svgui/configure.ac
--- sonic-visualiser-1.8/svgui/configure.ac.raptor2	2011-04-08 10:56:44.000000000 -0500
+++ sonic-visualiser-1.8/svgui/configure.ac	2011-07-31 20:33:13.936534473 -0500
@@ -82,7 +82,7 @@ SV_MODULE_REQUIRED([samplerate],[sampler
 SV_MODULE_REQUIRED([vamp],[vamp >= 2.1],[vamp/vamp.h],[],[])
 SV_MODULE_REQUIRED([vamphostsdk],[vamp-hostsdk >= 2.0],[vamp-hostsdk/PluginLoader.h],[],[])
 SV_MODULE_REQUIRED([rubberband],[rubberband],[rubberband/RubberBandStretcher.h],[rubberband],[rubberband_new])
-SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
+#SV_MODULE_REQUIRED([raptor],[raptor >= 1.4.20],[raptor.h],[raptor],[raptor_start_parse])
 SV_MODULE_REQUIRED([rasqal],[rasqal >= 0.9.19],[rasqal/rasqal.h],[rasqal],[rasqal_new_world])
 SV_MODULE_REQUIRED([redland],[redland >= 1.0.10],[librdf.h],[rdf],[librdf_new_world])
 
